scss/partials/_job.scss

```diff
@@ -160,10 +160,38 @@
         }
     }
 
-    &__time-social-links {
-        @extend .post-hero__time-social-links;
-    }
+    &__social-icon {
+        @include self() {
+            @include clear-decoration();
+            display: flex;
+            height: 6rem;
+            margin-left: auto;
+            width: 8rem;
+            
+            >*:not(:last-child) {
+                margin-right: 1rem;
+              }
+            
+          }
+
+          &__link {
 
+            &--linkedin {
+              mask: url("data:image/svg+xml;charset=utf8,<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 18 17.9'><path d='M12.7 15.3h2.7v-4.7c0-2.3-.5-4.1-3.2-4.1-1 0-2 .5-2.5 1.4V6.7H7v8.6h2.7V11c0-1.1.2-2.2 1.6-2.2 1.4 0 1.4 1.3 1.4 2.3v4.2zM2.4 4c0 .9.7 1.6 1.6 1.6S5.6 4.9 5.6 4 4.9 2.5 4 2.5 2.4 3.2 2.4 4zm.3 2.7v8.6h2.7V6.7H2.7zM1.3 0h15.3c.8 0 1.4.6 1.4 1.3v15.3c0 .7-.6 1.3-1.3 1.3H1.3c-.7 0-1.3-.6-1.3-1.3V1.3C0 .6.6 0 1.3 0z' fill-rule='evenodd' clip-rule='evenodd'/></svg>") no-repeat center / contain;
+              background-color: $secondary-green-2;
+            }
+        
+            &--twitter {
+              mask: url("data:image/svg+xml,%3Csvg version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' viewBox='-848.7 -103.7 1124.1 1000' enable-background='new -848.7 -103.7 1124.1 1000' xml:space='preserve'%3E%3Cpath d='M275.4,14.6c-41.6,20.4-85.8,33.7-132.7,39.8C190.5,23.2,227.1-26.1,244.5-85c-44.7,28.8-94.1,49.3-146.8,61.4c-42-49.3-102.1-80-168.2-80c-127.4,0-230.6,112.9-230.6,252.5c0,19.6,2,38.9,5.8,57.2C-486.6,196-656.8,95.1-770.6-57.3c-19.7,37.2-31.1,80.5-31.1,126.9c0,87.4,40.7,165.1,102.7,209.7c-37.9-1.6-73.5-12.3-104.5-31.4c0,1,0,2.3,0,3.4c0,122.1,79.5,224,185,247.5c-19.3,5.8-39.8,8.4-60.9,8.4c-14.8,0-28.9-1-43.1-4.5c29.2,100.7,114.4,173.8,215.4,175.3c-78.9,68-178.3,108-286.4,108c-18.7,0-37.1-1.1-55.1-3.5c102,71.9,223,113.8,353.5,113.8c424.1,0,656.2-384.7,656.2-718.3c0-10.6-0.2-21.5-0.8-32.3C205.3,109.6,244.5,65.1,275.4,14.6z'/%3E%3C/svg%3E%0A") no-repeat center / contain;
+              background-color: $secondary-green-2;
+            }
+        
+            &--facebook {
+              mask: url("data:image/svg+xml,%3Csvg width='19' height='18' viewBox='0 0 19 18' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M9.43472 0C4.46363 0 0.434723 4.05345 0.434723 9.05482C0.434723 13.5752 3.72535 17.3209 8.02847 18V11.6722H5.74332V9.05482H8.02847V7.05993C8.02847 4.79092 9.37144 3.53704 11.4281 3.53704C12.4125 3.53704 13.4425 3.71389 13.4425 3.71389V5.94223H12.307C11.189 5.94223 10.841 6.64079 10.841 7.35704V9.05482H13.3371L12.938 11.6722H10.841V18C15.1441 17.3209 18.4347 13.5752 18.4347 9.05482C18.4347 4.05345 14.4058 0 9.43472 0Z'/%3E%3C/svg%3E%0A") no-repeat center / contain;
+              background-color: $secondary-green-2;
+            }
+          }
+    }
 
```

Comment:
We indeed do have to refactor those into a separate file. But I'm guessing that is not going to happen now.